src/utils/logger/logger.ts

```diff
@@ -20,18 +17,16 @@ if (process.env.NODE_ENV === 'prod') {
       datePattern: 'YYYY-MM-DD',
       filename: process.env.LOG_FILE_NAME || 'brain-service-%DATE%.log',
       dirname: process.env.LOG_FILE_DIR || '.',
-    }),
+    })
   );
 }
 
-export const log =
-  (severity: SeverityLevel = SeverityLevel.INFO, logMessage: any, error?: any) => {
-    const errorDetails = error ?
-      { error: { message: error.message, stack: error.stack, name: error.name } } : {};
+export const log = (severity: SeverityLevel = SeverityLevel.INFO, logMessage: any, error?: any) => {
+  const errorDetails = error ? { error: { message: error.message, stack: error.stack, name: error.name } } : {};
```

Comment:
TSLint failed with this line (because it too long), I think the previous change was fine.

Reply:
done